### ISSUE TITLE
Still getting error in dlib shape detector. it is some problem of boost python .please someone help.I have tries all solutions posted in internet earlier but nothing came to help.

### ISSUE THREAD

**Issue by PaRthVaghela** [davisking/dlib#627]
pose_predictor = dlib.shape_predictor(predictor_model)
Boost.Python.ArgumentError: Python argument types in
    shape_predictor.__init__(shape_predictor, file)
did not match C++ signature:
    __init__(boost::python::api::object, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)
    __init__(_object*)

**Comment by davisking** [comment]
Are you running dlib's face_landmark_detection.py example program exactly as is without any modifications?

**Comment by PaRthVaghela** [comment]
Yes that's correct.i am running dlib's face_landmark_detection.py example program exactly as is without any modifications.

**Comment by davisking** [comment]
Then something is wrong with your boost python install. see: http://dlib.net/faq.html#Whywon'tthePythonbindingscompilework

**Comment by PaRthVaghela** [comment]
I'm using python2.7. I have python3.5 alongside with my os. I'm not using Anaconda still I got this problem. While installing boost python again, I got this : 
`Building Boost.Build engine with toolset gcc... tools/build/src/engine/bin.linuxx86_64/b2
Detecting Python version... 2.7
Detecting Python root... /usr
Unicode/ICU support for Boost.Regex?... not found.
Backing up existing Boost.Build configuration in project-config.jam.1
Generating Boost.Build configuration in project-config.jam...

Bootstrapping is done. To build, run:

    ./b2
    
To adjust configuration, edit 'project-config.jam'.
Further information:

   - Command line help:
     ./b2 --help
     
   - Getting started guide: 
     http://www.boost.org/more/getting_started/unix-variants.html
     
   - Boost.Build documentation:
     http://www.boost.org/build/doc/html/index.html
`
Your Help would be much appreciated.
